code/modules/events/aurora_caelus.dm

```diff
@@ -28,35 +28,56 @@
 
 /datum/round_event/aurora_caelus/start()
 	for(var/area in GLOB.sortedAreas)
-		var/area/A = area
-		if(A.area_flags & AREA_USES_STARLIGHT)
-			for(var/turf/open/space/S in A)
-				S.set_light(S.light_range * 3, S.light_power * 0.5)
+		var/area/effected_area = area
+		if(effected_area.area_flags & AREA_USES_STARLIGHT)
+			for(var/turf/open/space/spess in effected_area)
+				spess.set_light(spess.light_range * 3, spess.light_power * 0.5)
+		if(istype(effected_area, /area/station/service/kitchen))
+			for(var/turf/open/kitchen in effected_area)
+				kitchen.set_light(1, 0.75)
+			if(!prob(1) && !SSevents.holidays?[APRIL_FOOLS])
+				continue
+			var/obj/machinery/oven/roast_ruiner = locate() in effected_area
+			if(roast_ruiner)
+				roast_ruiner.balloon_alert_to_viewers("oh egads!")
+				var/turf/ruined_roast = get_turf(roast_ruiner)
+				ruined_roast.atmos_spawn_air("plasma=100;TEMP=1000")
+			for(var/mob/living/carbon/human/seymour as anything in GLOB.human_list)
+				if(seymour.mind && istype(seymour.mind.assigned_role, /datum/job/cook))
```

Comment:
this is gonna play no matter where the Chef is?

Reply:
chefs have a sixth sense for a ruined roast